content/en/docs/reference/using-api/server-side-apply.md

```diff
@@ -196,7 +196,8 @@ as [YAML](https://yaml.org/), with the media type `application/apply-patch+yaml`
 Whether you are submitting JSON data or YAML data, use
 `application/apply-patch+yaml` as the `Content-Type` header value.
 
-All JSON documents are valid YAML.
+Most JSON documents are accepted as YAML by Kubernetes. 
+you can find more details on this [open issue](https://github.com/go-yaml/yaml/issues/797)
```

Comment:
Let's keep “All JSON documents are valid YAML” as this remains true.

```suggestion
All JSON documents are valid YAML. However, Kubernetes has a bug where it uses a YAML
parser that does not fully implement the YAML specification. Some JSON escapes may
not be recognized.
```
We should link to a bug report in https://github.com/kubernetes/kubernetes; the _Kubernetes_ bug is partly down to our choice of parser library. (If we switched to a different library, the problematic behavior might go away).

Reply:
done.